src/LeftNav/LeftNav.less

```diff
@@ -68,7 +74,7 @@
   // In order to have a background image (right arrow) and a background color
   // simultaneously, they both must be written in the same CSS rule and the
   // image must come second.
-  .tint(background, @primary_blue, 6);
+  .shade(background, @neutral_off_white, .15);
```

Comment:
q: (for future updates) do we need smaller shade steps maybe? also, will it be possible to prescribe a scale? then we could maybe have `.shade--1`, `shade--2`, etc.